### ISSUE TITLE
Consider to use JWT for auth.

### ISSUE THREAD

**Issue by sharok89** [atomiomi/clavy#30]

**Comment by atomiomi** [comment]
Do you mean this library: https://github.com/tymondesigns/jwt-auth ?

**Comment by sharok89** [comment]
No. I meant JWT in general,  it's not important about implementation now.

**Comment by atomiomi** [comment]
Ooh, Google gave me this link. Is this some kind of approach for auth? Can you send me a link?

**Comment by sharok89** [comment]
Tokens are good choice for the auth, because they are stateless, safe and mobile ready. [Here](https://scotch.io/tutorials/the-anatomy-of-a-json-web-token) you can find some general information about token itself and find some reference to another articles, implementation for `Node.js` in particular.

**Comment by atomiomi** [comment]
I got it but do we have some issues that can be solved only by jwt? Futhermore what disadvantages current implementation has?

**Comment by sharok89** [comment]
The current implementation is very good for the clavy unless we decide to make mobile app.  The main reason to integrate JWT for me is learning something new :)

**Comment by atomiomi** [comment]
Ok) you may send pull request but I'm afraid I can't give you any advice about implementation. Will you take this issue as the next task?

**Comment by sharok89** [comment]
No, I think I will take it in the near future.

**Comment by atomiomi** [comment]
And what the next task?

**Comment by atomiomi** [comment]
We should allocate active issues
